language-support/hs/bindings/README.md

```diff
@@ -21,21 +21,46 @@ You can find some usage examples
-this directory that you can run as follows: `./build_packages.sh
-TARGET_DIR`.  This will build the tarballs for all cabal packages and
-put them in `TARGET_DIR`. If `TARGET_DIR` does not already contain a
-`cabal.project` file, it will also create a `cabal.project` file that
-references the created tarballs so all dependencies should be resolved
-properly. You will also need to install gRPC 1.22, see
-https://github.com/grpc/grpc/blob/master/BUILDING.md for installation
-instructions. If you install it in a non-standard location, you need
-to adjust extra-lib-dirs and extra-include-dirs for
-grpc-haskell-core. The default `cabal.project` file contains an
-example of how to do this.
-
-The main package for the ledger bindings is called `daml-ledger`.
+
+# Using these bindings **outside of this repository**
+
+*Note: These instructions document the current process. We do plan make it simpler!*
+
+These bindings can be exported as a standalone Haskell package `daml-ledger` using `stack`. Currently, `stack` references the code directly in the `daml` repo. The only annoyance is that part of the Haskell code for `daml-ledger` is generated from `.proto` files, and this generation step must be performed using the `bazel` build. We plan to simplify this step by directly providing `daml-ledger` as a package on hackage. Instructions for working with the daml repo can be found here: https://github.com/digital-asset/daml
+
+As well as `stack`, the only other prerequisite is to have `grpc` installed. (`grpc` is not required to generate the `daml-ledger` package, but it is required to use it).
+
+We are currently using gRPC version `1.23.0`. To install `grpc`  requires building `grpc` from source (really!). See detailed instructions here: https://github.com/grpc/grpc/blob/master/BUILDING.md.
+
+In the instructions below we do `make install` for grpc despite the warning from the `grpc` build instructions:
+
+> *"WARNING: After installing with make install there is no easy way to uninstall, which can cause issues if you later want to remove the grpc and/or protobuf installation or upgrade to a newer version."*
+
+If you decide against the `make install`, or choose a different install location, you will need to adjust the settings of `extra-lib-dirs` and `extra-include-dirs` in your `stack.yaml` config.
+
+Also, in the instructions below we export the `daml-ledger` package to `/tmp` which matches the locat
+ion declared in the `stack.yaml` of the example application `nim`. If you export somewhere else, you will need to adapt your `stack.yaml`
+
+## Download and build grpc at version 1.23.0
+
+    git clone -b v1.23.0 https://github.com/grpc/grpc
+    cd grpc
+    git submodule update --init
+    make
+    make prefix=/usr/local/grpc install
+
+## Clone daml repo, and export the daml-ledger package
+
+    cd /tmp
+    git clone https://github.com/digital-asset/daml.git
+    cd daml
+    direnv allow
```

Comment:
```suggestion
```
The next line takes care of initializing the environment, there is no need to do both.

Reply:
I wasn't sure. Removed it now.